### DISCUSSION TITLE
Crash on startup (loading preview image of a submarine failed)

### DISCUSSION THREAD

**Discussion by Foirt** [FakeFishGames/Barotrauma#14889] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

We use the mod to change the generation of the world and the stations were missing constructors and deconstructors (in consequence of which accumulated a lot of resources) and finally sailed to the station at which there are means for crafting sitting 1.5 - 2 hours over the crafting game flew out, and of course there are no saves, except where we docked. Started crafting again and to avoid such a repetition saved at the station and all the things that were on the station are gone, and this is almost all the resources that we extracted, saving only one - the last. 
Not enough save button without leaving the lobby, lack of saving things on the station, if the player left them there and did not sail away from it.

Translated with DeepL.com (free version)

### Reproduction steps

1. Start the game in the multiplayer campaign.
2. 2 hours to process resources into something useful, shells for cannons, etc. 
And after that a crash is possible.  

### Bug prevalence

Just once

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.5.9.2 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by SomeRandomNoobKekeke** [comment]
`C:\Program Files (x86)\Steam\steamapps\common\Barotrauma\crashreport.log`? 
`C:\Program Files (x86)\Steam\steamapps\common\Barotrauma\servercrashreport.log`?

**Reply by Foirt** [reply]
[crashreport.log](https://github.com/user-attachments/files/17478628/crashreport.log)
servercrashreport missing in the folder

**Reply by SomeRandomNoobKekeke** [reply]
Are you sure this is the last one and there's no `crashreport (2).log`, `crashreport (100).log`?

**Reply by Regalis11** [reply]
Thank you for the report! The error messages in the crash report seem to indicate that some submarine files (both vanilla and modded) have broken somehow. [Verifying file integrity](https://help.steampowered.com/en/faqs/view/0C48-FCBD-DA71-93EB) might solve the issue. It's hard to say what caused them to get corrupted in the first place, but it could potentially be related to some mod.

This crash report also seems to be from last May, and from a much older version of the game. Are you sure it's the correct report?

**Reply by Foirt** [reply]
This is the only bug report in the game folder, there are no others
